File: src/main/java/ladanse/website/domain/LoginEvent.java
package ladanse.website.domain;

import javax.persistence.*;
import java.util.Date;

@Entity
@Table(name = "LoginEvent")
public class LoginEvent
{
   private Long id;
   private Date loginTime;
   private Account account;

   protected LoginEvent()
   {
   }

   public LoginEvent(Account account)
   {
	 setLoginTime(new Date());
	 setAccount(account);
   }

   @Id
   @Column(name = "id")
   @GeneratedValue(strategy = GenerationType.IDENTITY)
   public Long getId()
   {
	 return id;
   }

   protected void setId(Long id)
   {
	 this.id = id;
   }

   @Basic
   @Column(name = "loginTime")
   @Temporal(TemporalType.TIMESTAMP)
   public Date getLoginTime()
   {
	 return loginTime;
   }

   protected void setLoginTime(Date loginTime)
   {
	 this.loginTime = loginTime;
   }

   @ManyToOne
   @JoinColumn(name = "account", nullable = false)
   public Account getAccount()
   {
	 return account;
   }

   protected void setAccount(Account account)
   {
	 this.account = account;
   }
}

File: src/LaDanse/DomainBundle/Entity/LoginEvent.php
<?php

namespace LaDanse\DomainBundle\Entity;

use Doctrine\ORM\Mapping as ORM;


/**
 * @ORM\Entity
 * @ORM\Table(name="LoginEvent")
 */
class LoginEvent
{
    const REPOSITORY = 'LaDanseDomainBundle:LoginEvent';

    /**
     * @ORM\Column(type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    protected $id;

    /**
     * @ORM\Column(type="datetime", nullable=false)
     */
    protected $loginTime;

    /**
     * @ORM\ManyToOne(targetEntity="Account")
     * @ORM\JoinColumn(name="accountId", referencedColumnName="id", nullable=false)
     */
    protected $account;

    /**
     * Get id
     *
     * @return integer 
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * Set loginTime
     *
     * @param \DateTime $loginTime
     * @return LoginEvent
     */
    public function setLoginTime($loginTime)
    {
        $this->loginTime = $loginTime;

        return $this;
    }

    /**
     * Get loginTime
     *
     * @return \DateTime 
     */
    public function getLoginTime()
    {
        return $this->loginTime;
    }

    /**
     * Set account
     *
     * @param Account $account
     * @return LoginEvent
     */
    public function setAccount(Account $account = null)
    {
        $this->account = $account;

        return $this;
    }

    /**
     * Get account
     *
     * @return Account
     */
    public function getAccount()
    {
        return $this->account;
    }
}
